compiler/control/CompileMethod.cpp

```diff
@@ -21,11 +21,15 @@
 
 #include "control/CompileMethod.hpp"
 
+#if defined(OMR_OS_WINDOWS)
+#include <process.h>                           // for _getpid
```

Comment:
Just validating - our minimum compiler version is VS2010 for Windows IIRC. Is `_getpid` supported there? I haven't been able to find the docs for it but it looks like it should be.

Reply:
Did a bunch of digging. It looks like it is supported. The `_getpid()` call will boil down to a `GetCurrentProcessId()` WIN_API even on VS2010. I think we're good here.